es6-arrow-functions/main.js

```diff
@@ -0,0 +1,51 @@
+const $jokeContainer = document.querySelector('#joke-container');
+const $flashImage = document.querySelector('#flash-image');
+const $jokeForm = document.querySelector('#joke-form');
+
+const jokester = {
+
+  tellJoke: (setup, punchline) => {
+    this.renderJokePhrase = jokester.renderJokePhrase;
+    this.appendJokePhrase = jokester.appendJokePhrase;
```

Comment:
If you don't convert this `tellJoke` method to an arrow function, you won't have to do this weird reassignment thing.